File: Programming-languages-and-laboratory/Python/Exercises/21b_compito_java_20180406/Normale.py
from Giocatore import Giocatore


class Normale(Giocatore):
    def __init__(self, codice, tipo, nome_cognome, circolo, vinti, disputate):
        super().__init__(codice, tipo, nome_cognome)
        self._circolo = circolo
        self._vinti = vinti
        self._disputate = disputate

    def __str__(self):
        return super().__str__() + "\t-\t-\t-\t" + self._circolo + "\t" + str(self._vinti) + "\t" + str(
            self._disputate)

File: Programming-languages-and-laboratory/Java/Exercises/21_compito_java_2018_04_06b/Normale.java
public class Normale extends Giocatore {
	private String nomeCircolo;
	private int setVinti;
	private int partiteDisputate;

	public Normale(int codiceGiocatore, String tipo, String nomeCognome, String nomeCircolo, int setVinti,
			int partiteDisputate) {
		super(codiceGiocatore, tipo, nomeCognome);
		this.nomeCircolo = nomeCircolo;
		this.setVinti = setVinti;
		this.partiteDisputate = partiteDisputate;
	}

	public String toString() {
		return nomeCognome + "\t" + codiceGiocatore + "\t" + tipo + "\t-\t-\t-\t" + nomeCircolo + "\t" + setVinti + "\t"
				+ partiteDisputate;
	}
}